wallet/src/wallet.rs

```diff
@@ -391,6 +392,29 @@ pub fn parse_command(
 
 type ProcessResult = Result<String, Box<dyn error::Error>>;
 
+fn check_account_for_fee(
+    rpc_client: &RpcClient,
+    config: &WalletConfig,
+    lamports_per_signature: u64,
+) -> Result<(), Box<dyn error::Error>> {
+    check_account_for_multiple_fees(rpc_client, config, lamports_per_signature, 1)
+}
+
+fn check_account_for_multiple_fees(
+    rpc_client: &RpcClient,
+    config: &WalletConfig,
+    lamports_per_signature: u64,
+    tx_count: u64,
```

Comment:
nit: change from `tx_count` to `signature_count`

Reply:
Obviated by the change below